packages/server/app/Controllers/Http/MpeRoomsHttpController.ts

```diff
@@ -15,13 +15,15 @@ const MPE_ROOMS_SEARCH_LIMIT = 10;
 export default class MpeRoomsHttpController {
     public async listAllRooms({
         request,
+        bouncer,
         auth,
     }: HttpContextContract): Promise<ListAllMpeRoomsResponseBody> {
         const user = auth.user;
         invariant(
             user !== undefined,
             'User must be logged in to list all Mpe rooms',
         );
+        await bouncer.authorize('hasConfirmedEmail');
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Bouncer package is really neat 👌